assets/styles/1-design-tokens/_variables--spacing.scss

```diff
@@ -1,7 +1,7 @@
 /* Create the Global Spacing CSS Variables */
 // Base = 8px (Spacing for mobile)
 :root{
-    --space-unit: 0.5;
+    --space-unit: 1.0;
```

Comment:
So I found that none of the spacing was accurately described unless the space unit was 1.0.  So I corrected it.

If I'm missing something here, please correct me.  But this allowed me to trust the comments on the side about how large/small things were.

Reply:
@Gmfholley I *think* it was set up for base 8px or .5 on mobile and then 16px or 1 on desktop, but if that's not what's happening could you show me for my own knowledge and learning. Thanks!

Reply:
```suggestion
    --space-unit: 0.5;
```